routes/auth.js

```diff
@@ -71,15 +73,54 @@ router.get('/github/finalize', passportOrPat(),
     // user's token to any site that asks for it. see:
     // https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage
     res.status(200).send(`<script>
-      window.opener.postMessage({
-        type: 'github-token',
-        token: '${safeToken}',
-      }, '${origin}');
+      window.opener.postMessage(${result}, '${origin}');
       window.close();
     </script>`);
   }
 );
 
+/**
+ * Fetch a list of ClearlyDefined permissions related to the given token in the identified org.
+ * @param {string} token - GitHubApi token
+ * @param {string} org - org name to filter teams
+ * @returns {Promise<Array<string>>} - list of permission names
+ */
+async function getPermissions(token, org) {
+  const options = { headers: { 'user-agent': 'clearlydefined.io' } };
+  const client = new GitHubApi(options);
```

Comment:
low-pri, sometime we'll want to consolidate the client creations. partially my fault. :)